hello-world-master/source/components/scene/itemcontentnode.xml

```diff
@@ -0,0 +1,7 @@
+<component  name = "itemContentNode" extends = "ContentNode" >
+
+  <interface >
+    <field id = "image_1080_url" type = "string" />
```

Comment:
Same indentation as mentioned in hello-world-master/source/components/scene/descriptionscreen.xml